hazelcast/src/main/java/com/hazelcast/util/executor/StripedExecutor.java

```diff
@@ -149,7 +149,15 @@ private Worker getWorker(Runnable command) {
         return workers[index];
     }
 
-    // used in tests.
+    public List<BlockingQueue<Runnable>> getWorkQueues() {
```

Comment:
I'm not super happy about exposing this, but it's the easiest way to access the queued events.